style.css

```diff
@@ -0,0 +1,534 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
- [ ] Good job using the correct background as requested by the previewer :clap: , kindly make sure to fix your mobile version too to complete this milestone :recycle: 

Your mobile version:

![rawcdn githack com_mhnazary_Desktop-_Version_Portfolio_1adc458365278ac8cb0eeddbab68f28858e7cc61_index html](https://user-images.githubusercontent.com/97834160/230707939-7b832194-041f-476d-a2ec-83d7351ea573.png)